docs/photoniq/prerendering/best-practices-prerendering.md

```diff
@@ -33,31 +39,8 @@ Customize your origin server to detect user-agent strings. This enables more gra
 
 Examine the performance metrics generated by the prerendering service. These metrics provide valuable insights into user experience and identify areas for further optimization. Make a habit of checking these metrics at regular intervals to stay ahead of any issues.
```

Comment:
Can we give more guidance here? Like suggest specific metrics they should pay extra attention to?

Reply:
Area for further opt will come with SEO scoring
right now, They can only monitor the overall health of the system 
Example:
Response Time Percentiles
Failed Renders